app/javascript/components/charts/lollipop-chart/component.jsx

```diff
@@ -38,6 +38,16 @@ class LollipopChart extends PureComponent {
           : selectedUnitConfig.value;
     }
 
+    // TODO: pass as prop ?
+    const customFormat = ({ num, roundTo }) => {
+      if (num === 0) return '0';
+      let number = format(`.${roundTo || 2}f`)(num);
+      if (number.charAt(number.length - 1) === '0') {
```

Comment:
Im not sure I understand what is happening here. Can you explain and give some examples?

Reply:
Sure! It just removes trailing zeros i.e. 0.70 to 0.7.

Reply:
Ah I see it now. Thanks.